packages/run-protocol/test/psm/test-psm.js

```diff
@@ -0,0 +1,190 @@
+  return contractBundle;
+};
+
+const BASIS_POINTS = 10000n;
+const WantStableFeeBP = 1n;
+const GiveStableFeeBP = 3n;
+const MINT_LIMIT = 20_000_000n * 1_000_000n;
+
+/**
+ * Compute the fee in runBrand based on `given`. Choose the fee ratio
+ * appropriate to teh `given` brand.
+ *
+ * @param {Amount<'nat'>} given
+ * @param {Brand} stableBrand
+ * @returns {Amount<'nat'>}
+ */
+const minusFee = (given, stableBrand) => {
+  const feeBP = given.brand === stableBrand ? GiveStableFeeBP : WantStableFeeBP;
+  return AmountMath.make(
+    stableBrand,
+    BigInt((given.value * (BASIS_POINTS - feeBP)) / BASIS_POINTS),
+  );
+};
+
+// let testJig;
+const setJig = _jig => {
+  // testJig = jig;
+};
+
+/**
+ * @param {ExecuteContract} t
+ */
+test.before(async t => {
+  // makeBundle is slow, so we bundle each contract once and reuse in all tests.
+  const [psmBundle] = await Promise.all([makeBundle(psmRoot)]);
+  t.context.bundles = { psmBundle };
+  const { makeFar, makeNear: makeRemote } = makeLoopback('zoeTest');
+  const { zoeService, feeMintAccess: nonFarFeeMintAccess } = makeZoeKit(
+    makeFakeVatAdmin(setJig, makeRemote).admin,
+  );
+  /** @type {ERef<ZoeService>} */
+  const zoe = makeFar(zoeService);
+  t.context.zoe = zoe;
+  trace('makeZoe');
+  const feeMintAccess = await makeFar(nonFarFeeMintAccess);
+  t.context.feeMintAccess = feeMintAccess;
+
+  const runIssuer = await E(zoe).getFeeIssuer();
+  const runBrand = await E(runIssuer).getBrand();
+  t.context.runKit = { runIssuer, runBrand };
+  const anchorKit = makeIssuerKit('aUSD');
+  t.context.anchorKit = anchorKit;
+
+  const { brand: anchorBrand } = anchorKit;
+  const psmInstall = await E(zoe).install(psmBundle);
+  t.context.installs = { psmInstall };
+  const mintLimit = AmountMath.make(anchorBrand, MINT_LIMIT);
+  t.context.mintLimit = mintLimit;
+  t.context.terms = {
+    anchorBrand,
+    main: { WantStableFeeBP, GiveStableFeeBP, MintLimit: mintLimit },
+  };
+});
+
+test('simple trades', async t => {
+  const {
+    zoe,
+    feeMintAccess,
+    terms,
+    installs: { psmInstall },
+    runKit: { runIssuer, runBrand },
+    anchorKit: { brand: anchorBrand, issuer: anchorIssuer, mint: anchorMint },
+  } = t.context;
+  const { publicFacet } = await E(zoe).startInstance(
+    psmInstall,
+    harden({ AUSD: anchorIssuer }),
+    terms,
+    harden({ feeMintAccess }),
+  );
+  const give = AmountMath.make(anchorBrand, 200n * 1_000_000n);
+  const seat1 = await E(zoe).offer(
+    E(publicFacet).makeSwapInvitation(),
+    harden({ give: { In: give } }),
+    harden({ In: anchorMint.mintPayment(give) }),
+  );
+  const runPayout = await E(seat1).getPayout('Out');
+  const expectedRun = minusFee(give, runBrand);
+  const actualRun = await E(runIssuer).getAmountOf(runPayout);
+  t.deepEqual(actualRun, expectedRun);
+  const liq = await E(publicFacet).getCurrentLiquidity();
+  t.true(AmountMath.isEqual(liq, give));
+  trace('get stable', { give, expectedRun, actualRun, liq });
```

Comment:
I think once the test is working, this should be dropped.  (Unless we're going to give `trace` a toggle.)

Reply:
disabled them